test/io.jl

```diff
@@ -80,3 +82,17 @@ CSV.write(genDataPath*"/daf.csv", daf.fcstable)
         end
     end
 end
+
+@testset "Checksums" begin
```

Comment:
May you use a `.json` file for all the checksums instead of so many `.txt` files?

Reply:
but to use `.json` we have to use a new dep `using JSON` correct?
I can do it if you want

Reply:
done